airflow/kubernetes/pod_launcher.py

```diff
@@ -33,13 +33,21 @@
 from requests.exceptions import BaseHTTPError
 from .kube_client import get_kube_client
 
+# Used to calculate semantic versioning
+try:
+    from packaging.version import parse as semantic_version
+except ImportError:
+    # Python 2
+    from distutils.version import LooseVersion as semantic_version
```

Comment:
drop py 2 support?

Reply:
Yeah. Drop it for the master branch. We can worry about that for cherry-picking if we do another 1.10 airflow.